react-secure-biometric-interface-integrator/README.md

```diff
@@ -25,14 +25,13 @@ npm publish
 ## Component
 
 ```js
-<MosipBioDevice 
+<SecureBiometricInterfaceIntegrator 
     transactionId="your_transaction_id"
-    labelName="label_name"
-    buttonName="button_name"
-    biometricEnv={biometricEnv}
+    buttonLabel="button_label"
+    langCod="en"
```

Comment:
it should be langCode

Reply:
added in this [0d71812](https://github.com/mosip/mosip-plugins/pull/58/commits/0d71812a3dfb6f5b38806a36638a1e7bc0401412)